File: lab6.5/lab6.5-1.py
list1 = [1, 2, 3, 4, 5]
list2 = [3, 4, 5, 6, 7]


def common_elements(list1, list2):
    return [elem for elem in list1 if elem in list2]


common = common_elements(list1, list2)
print(common)

File: lab6.5/lab6_5_1.cpp
#include <iostream>
using namespace std;

void find_common_elements(int list1[], int size1, int list2[], int size2) {
    cout << "Common elements are: ";
    for (int i = 0; i < size1; i++) {
        for (int j = 0; j < size2; j++) {
            if (list1[i] == list2[j]) {
                cout << list1[i] << " ";
                break;
            }
        }
    }
    cout << endl;
}
